tests/test_projectbuilder.py

```diff
@@ -111,6 +130,15 @@ def test_init(mocker, test_flit_path, legacy_path, test_no_permission, test_bad_
         build.ProjectBuilder(test_bad_syntax_path)
 
 
+@pytest.mark.parametrize('value', [b'something', typing.Text('something_else')])
+def test_python_executable(test_flit_path, value):
```

Comment:
You test the happy path, but not the sad path? IMHO we should do both.

Reply:
I removed the checks.